### ISSUE TITLE
Find Places in offline search by short_name

### ISSUE THREAD

**Issue by paskalito** [osmandapp/OsmAnd#8412]
### tl;dr
 be really super handy if things/places could be found by their short_name ( maybe also alt_name, old_name )

### long story /background
Because in my case i regularly try navigate to a place that has a really long annoying full name to type (with many common words) but a very nice short name by which is uniqu and also everybody is calling it like that

I unknowingly added this places short_name into te to the name but then someone in the community put it into short_name. - that is tecnically correct from the osm side of things but totally annoying cause i still can't find my place !!! (conveniently)

Also i think this is only releveant for offline because now that i am home and in wlan pressing the search online button it got found immedeatly..

is somehow slightly connected with this https://github.com/osmandapp/Osmand/issues/6371 more that https://github.com/osmandapp/Osmand/issues/6371#issuecomment-448891283

**Comment by vshcherb** [comment]
Places should be already searchable by alt_name and old_name.

**Comment by paskalito** [comment]
well so i bug report my case:

osm element:
https://www.openstreetmap.org/node/4946716109/history

short_name is: Vuesch e.V. at least since 2 years.

I have osmand offline maps of berlin state February 2020.

if i search in osmand. i don't get results for:
- vuesch
- Vuesch
- Vuesch e.V.
- VUESCH

if i edit the poi within osmand i can see the short_name correctly. ( i even just copy pasted it from there and didn't get a result)

same situation with 
https://www.openstreetmap.org/way/286547803

and searching for
- bfs
- BfS

**Comment by arbitrary-dev** [comment]
+ support for `short_name:en` and other localizations

**Comment by zekooooo** [comment]
In some languages with possessive adjectives, long and short forms of street names are usually more different than in English. Example in Croatian: "Ulica Nikole Škrlca" and "Škrlčeva". Searching for the short form here will only match the long form if you stop several letters before the end of the world. This may not be obvious to the user and/or the search string could be too general and match a lot of stuff.

**Comment by Zirochkabila** [comment]
# Testing 1

> OsmAnd Maps 5465, released 16.09.26
> OsmAnd~ 5.5.0#8833m, released: 2026-09-16

| # | Search Engine | Query | Target Object | Expected Result | Actual Result | Status |
| :--- | :--- | :--- | :--- | :--- | :--- | :---: |
| **1a** | **Standard Search** *(iOS / Android)* | `vuesch` / `Vuesch e.V.` | Node 4946716109 | Found via `short_name` tag | Object is found in top results | 🟢 **PASSED** |
| **1b** | **Spatial Search** *(Android)* | `vuesch` / `Vuesch e.V.` | Node 4946716109 | Found via `short_name` tag | Object is found in top results | 🟢 **PASSED** |
| **2a** | **Standard Search** *(iOS / Android)* | `bfs` / `BfS` | Way 286547803 | Found via `short_name` tag | Object is found in top results | 🟢 **PASSED** |
| **2b** | **Spatial Search** *(Android)* | `bfs` / `BfS` | Way 286547803 | Found via `short_name` tag | Object is found in top results | 🟢 **PASSED** |
